platform/features/layout/src/FixedController.js

```diff
@@ -21,8 +21,22 @@
  *****************************************************************************/
 
 define(
-    ['./FixedProxy', './elements/ElementProxies', './FixedDragHandle'],
-    function (FixedProxy, ElementProxies, FixedDragHandle) {
+    [
+        'lodash',
+        './FixedProxy',
+        './elements/ElementProxies',
+        './FixedDragHandle',
+        '../../table/src/TelemetryCollection',
+        '../../../../src/api/objects/object-utils'
```

Comment:
Yup, definitely some better organization needed here. In the case of TelemetryCollection, here are some possibilities -
* `src/telemetry/TelemetryCollection`
* `src/util/TelemetryCollection`

Neither of these directories exist currently. I have a slight preference for `src/telemetry`, but a telemetry directory would be empty apart from the `TelemetryCollection`. I think there are some other source files (like `object-utils.js`) that could be placed in a `util` directory however. I could be persuaded either way.

Something that I've thought about with object-utils is that it would be neat if it could somehow be exposed via the `AdapterCapability`. The functions could be automatically bound to the `domainObject` that the capability is associated with.